metadata.yaml

```diff
@@ -23,5 +23,6 @@ storage:
     type: filesystem
     minimum-size: 1M
 
-assumes:
-  - k8s-api
```

Comment:
We do not assume `k8s`?

Reply:
This was lost during the rebase it seems. Just added it back.